app/templates/reminders/reminder/edit.hbs

```diff
@@ -1 +1 @@
-{{edit-reminder model=model action='editReminder' action='rollbackChanges'}}
+{{edit-reminder model=model edit=true action='editReminder' action='rollbackChanges'}}
```

Comment:
So right now it looks like you're triggering edit to be true as soon as you click the edit button. It makes more sense to have the indication pop up once you actually start changing a field (i.e. when there are dirty attributes). If a user hasn't actually made any changes they shouldn't be flagged to save unsaved changes.